src/main/java/com/myfirstownapp/beeapp/harvest/HarvestRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.myfirstownapp.beeapp.harvest
+
+import org.springframework.data.repository.CrudRepository
+
+interface HarvestRepository : CrudRepository<Harvest, Long> {
+    override fun findAll(): List<Harvest>
```

Comment:
it's unnecessary, same as `override fun <s: Harvest?> save(entity:S):S` Do you know why?